ownCloud/Viewer/OCExtension Subclasses/OCDisplayExtension+DisplayView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  OCExtension+DisplayView.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 30/08/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import ownCloudSDK
+
+extension OCDisplayExtension {
+
+	static func normalPDFExtension(identifier: String) -> OCDisplayExtension {
```

Comment:
This static/class method could be moved to the `PDFViewerViewController` class method/property `extension` to make it more self-contained.